packages/preset-rules/src/customRules.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as fs from 'fs';
+import path from 'path';
+import axe from 'axe-core';
+
+// Directory containing custom rules and checks
+const customRulesPathDir = 'custom-rules';
+
+export function registerCustomRules(): void {
+    const directoryPath = path.join(__dirname, customRulesPathDir);
+    const newChecks: axe.Check[] = [];
+    const newRules: axe.Rule[] = [];
+
+    try {
+        // Path to the JSON file with changes to existing rules
+        const changesInExistingRulesPath = path.join(directoryPath, 'changes.json');
```

Comment:
what does `changes.json` file nomenclature for? Does it contain customized changes to rules and checks?

Reply:
changes.json nomenclature is for having any changes in the existing rules. [Reference](https://docs.deque.com/devtools-for-web/4/en/cli-rulesets#:~:text=directory%0A%20%E2%94%9C%20changes.json%0A%20%E2%94%9C%20rules%0A%20%E2%94%82%20%20%E2%94%94%20h1%2Dno%2Dduplicate.json%0A%20%E2%94%94%20checks%0A%20%20%20%20%E2%94%94%20page%2Dno%2Dduplicate%2Dh1.json)